src/Execution/Executor.php

```diff
@@ -217,8 +218,8 @@ public function executeFieldsSerially(
 
         $promise->then(function ($resolvedResults) use (&$finalResults) {
             $finalResults = $resolvedResults ?? [];
-        })->otherwise(function ($ex) {
-            $this->context->addError($ex);
+        })->otherwise(function (ExecutionException $ex) {
```

Comment:
What if `$ex` is not `ExecutionException`? Do we lose that exception?

Reply:
Sorry, clicked the wrong button. I'm also wondering if this is correct?

Reply:
You're right, I've fixed this now.